encoder.go

```diff
@@ -117,7 +117,6 @@ func EncodeFloat64(w io.Writer, f float64) (err error) {
 func EncodeTime(w io.Writer, t time.Time) (err error) {
 	var s = t.Unix()
 	var ns = int32(t.Nanosecond()) // this int64 -> int32 is safe.
-
```

Comment:
Any particular reason to remove the whitespace here?

Reply:
nope... I think, I've added some debugging code which I deleted later (and with it that innocent line). Can reintroduce that.